rootfs/opt/fluentd/deis-output/lib/fluent/plugin/out_deis.rb

```diff
@@ -0,0 +1,66 @@
+require 'fluent/mixin/config_placeholders'
+require 'fluent/mixin/plaintextformatter'
+require 'fluent/mixin/rewrite_tag_name'
+require 'fluent/mixin/deis'
+require 'fluent/output'
+
+module Fluent
+  class DeisOutput < Output
+    Fluent::Plugin.register_output("deis", self)
+
+    include Fluent::Mixin::PlainTextFormatter
+    include Fluent::Mixin::ConfigPlaceholders
+    include Fluent::HandleTagNameMixin
+    include Fluent::Mixin::RewriteTagName
+    include Fluent::Mixin::Deis
+
+    config_param :tag, :string, :default => ""
+    config_set_default :output_include_time, false
+    config_set_default :output_include_tag, false
+    config_set_default :num_threads, 5
+    config_set_default :flush_thread_count, 5
+
+    def initialize
```

Comment:
Glancing at docs on creating fluentd plugins, I am wondering if, perhaps, it might be more conventional (from fluentd's perspective [that is to say not from a pure Ruby perspective]) to define a method `def configure(conf)` here instead of an initializer.

http://docs.fluentd.org/articles/plugin-development#writing-non-buffered-output-plugins